lib/grpnames.gd

```diff
@@ -73,24 +93,24 @@
 ##  Remark:
 ##   The normal subgroup lattice of G does not contain
 ##   all the information needed for the normal subgroup lattice
-##   of G2 for a decomposition G = G1 x G2. However let 
+##   of G2 for a decomposition G = G1 x G2. However let
 ##   G = G1 x .. x Gr be finest direct product decomposition
 ##   then all Gi are normal subgroups in G. Thus all Gi occur in
-##   the set NormalSubgroups(G) and we may split G recursively 
+##   the set NormalSubgroups(G) and we may split G recursively
 ##   without recomputing normal subgroup lattices for factors.
 ##
 ##  Method to enumerate factorizations given the divisors:
 ##   Consider a strictly increasing chain
 ##     1  <  a1 < a2 < .. < a_n  <  A
-##   of positive divisors of an integer A. 
+##   of positive divisors of an integer A.
 ##   The task is to enumerate all pairs 1 <= i <= j <= n
 ##   such that a_i*a_j = A. This is done by
 ##
-##   i := 1;  
+##   i := 1;
 ##   j := n;
-##   while i <= j do                   
-##     while j > i and a_i*a_j > A do 
-##       j := j-1; 
+##   while i <= j do
+##     while j > i and a_i*a_j > A do
+##       j := j-1;
 ##     end while;
```

Comment:
Regarding code formatting conventions: This is a bit problematic, because whilere there is a semi-official standard (namely the way GAP prints functions back to if you ask it -- try e.g. `Display(Intersection);`), a lot of GAP library code deviates from that. We actually plan to discuss that on the next GAP days in January (see https://github.com/gapdays/gap-sage-days2016/wiki/Suggestions-for-the-agenda)

For now, it is probably a good idea to just match the immediately surrounding code. This way, at least *inside* any given function the code is formatted consistently.